plugins/plugin-java/che-plugin-java-ext-lang-client/src/test/java/org/eclipse/che/ide/ext/java/client/project/classpath/valueproviders/pages/sources/SourceEntryPresenterTest.java

```diff
@@ -58,8 +58,8 @@
  */
 @RunWith(GwtMockitoTestRunner.class)
 public class SourceEntryPresenterTest {
-    final private static String PLAIN_TYPE = "plainJava";
-    final private static String SOURCE     = "lib";
+    final private static String JAVAC  = "javac";
```

Comment:
it declared in other module, so for now o want to keep it as is after release we will fix this